web/react/components/create_comment.jsx

```diff
@@ -105,14 +106,17 @@ module.exports = React.createClass({
             this.lastTime = t;
         }
     },
-    handleUserInput: function(messageText) {
+    handleUserInput: function(message) {
+        var messageText = CreatePost.truncateText(message);
```

Comment:
I'm 0/5, just didn't really know where else to put it, I know we wanted to avoid putting more stuff in utils.jsx if there was no chance it would be used elsewhere and I didn't think making a separate "create_message_utils.jsx" file or something similar would be good.  If there's a better way to handle it I'm all for it, but I figured this is the method we were moving towards.